model/transaction/_meta/fields.yml

```diff
@@ -221,6 +254,50 @@
               Version of the framework used.
             overwrite: true
 
+        - name: origin
```

Comment:
Can we limit this to only the fields that we'll set? i.e. id, name, and version

Reply:
oops! yes, I forgot to change this when I updated the code